optional_plugins/resultsdb/avocado_resultsdb/__init__.py

```diff
@@ -142,6 +143,21 @@ def _status_map(status):
         return 'NEEDS_INSPECTION'
 
 
+class ResultsdbResult(Result):
+
+    """
+    ResultsDB render class
+    """
+
+    name = 'resultsdb'
+    description = 'Resultsdb result support'
+
+    def render(self, result, job):
+        if getattr(job.args, 'resultsdb_logs', None) is not None:
+            LOG_UI.info("JOB URL    : %s/%s" % (job.args.resultsdb_logs,
```

Comment:
It's been a while since I modified logging so I'm not 100% sure, but won't this emit the message even when non-human output is enabled? @dmonakhov can you please double-check this message does not spoil execution with other plugin enabled. This should not produce the `JOB URL` message and should only look like this:

```
avocado run --resultsdb ... --tap - -- /bin/true
1..1
# debug.log of /bin/true:
ok 1 /bin/true
```

If the `JOB URL` is there you ought to wrap it in:

```
            if getattr(job.args, 'stdout_claimed_by', None) is None:
```

conditional (viz html plugin, ...).

Reply:
(I don't have `resultsdb` and time to set it right now..)

Reply:
Yep, good catch: https://github.com/avocado-framework/avocado/pull/2321

Reply:
Yep... you are right
I'll send new PR

Reply:
@dmonakhov, please just review https://github.com/avocado-framework/avocado/pull/2321